### PR TITLE
[HER-83] Create camera entity

### PR DESCRIPTION
**Pull request by TBiannca**
## Description

- it will live inside data layer
- it will be used by our dbcontext
- it will have the following fields
    - id
    -latitude
    -longitude
-ensure migration is created beforehand
-add ECamera to dbcontext

## Related 

[HER-83](https://nerds-interns.atlassian.net/browse/HER-83?atlOrigin=eyJpIjoiNzVjZGNiNmIzZmU5NDMwYmI3MjY0ZmZlYjAwZjViYTciLCJwIjoiaiJ9)

### CODE REVIEW

File: src/Data/ECamera.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class ECamera
+    {
+        [Required]
+        public string Id { get; set; }
+
+        [Required]
+        public int Latitude { get; set; }
```
Comment (RaduBuciuceanu) [inline src/Data/ECamera.cs:11]: Shouldn't it be a string here?

File: src/Data/ECamera.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+        [Required]
+        public int Longitude { get; set; }
```
Comment (RaduBuciuceanu) [inline src/Data/ECamera.cs:14]: Shouldn't it be a string here?

File: src/Data/ECamera.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    }
+}
```
Comment (RaduBuciuceanu) [inline src/Data/ECamera.cs:16]: Should we move the new entity under the `Data/Camera/Entities` folder? Should we do the same with EDetection?

**Review by RaduBuciuceanu — CHANGES_REQUESTED** [review]

File: src/Data/ECamera.cs (hunk 1 above)
Comment (TBiannca) [reply src/Data/ECamera.cs]: done

File: src/Data/ECamera.cs (hunk 2 above)
Comment (TBiannca) [reply src/Data/ECamera.cs]: done

File: src/Data/ECamera.cs (hunk 3 above)
Comment (TBiannca) [reply src/Data/ECamera.cs:16]: done

**Review by andreea-dumitrascu — APPROVED** [review]

**Review by Urzica-Platon-Casian — APPROVED** [review]

**Review by RaduBuciuceanu — APPROVED** [review]

**Pull request merged**